src/test/scala/org/kibanaLoadTest/simulation/branch/DashboardJourney.scala

```diff
@@ -2,22 +2,22 @@ package org.kibanaLoadTest.simulation.branch
 
 import io.gatling.core.Predef._
 import io.gatling.core.structure.ScenarioBuilder
-import org.kibanaLoadTest.scenario.{Dashboard, Login}
+import org.kibanaLoadTest.helpers.KbnClient
+import org.kibanaLoadTest.scenario.Dashboard
 import org.kibanaLoadTest.simulation.BaseSimulation
 
 class DashboardJourney extends BaseSimulation {
   val scenarioName = s"DashboardJourney"
   props.maxUsers = 500
+  val client = new KbnClient(appConfig)
+  val cookiesLst = client.generateCookies(props.maxUsers)
+  val circularFeeder = Iterator
```

Comment:
Maybe pregenerate in a before hook?

Reply:
It turns out hooks are not part of Gatling DSL, so it can contain some independent API calls but not the objects related to session (cookies are actually the one we use in use session). I moved archives loading in before hook, but had to keep cookie generation as is